docker/sandbox/kustomization.yaml.tmpl

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Probably worth removing the .tmpl suffix in the filename since you are already doing so with the copy.